starknet_py/net/account/account_client.py

```diff
@@ -73,6 +75,34 @@ async def _get_nonce(self) -> int:
         )
         return nonce
 
+    async def get_balance(self, token_address: AddressRepresentation = None) -> int:
+        """
+        Checks account's balance of specified token._
+
+        :param token_address: Address of the ERC20 contract.
+                              If not specified it will be mainnet or testnet payment token address.
+        :return: Account's balance of token.
+        """
+        if token_address is None:
+            if self.net == MAINNET:
+                token_address = MAINNET_ETH_CONTRACT
+            elif self.net == TESTNET:
+                token_address = TESTNET_ETH_CONTRACT
+            else:
+                raise BadRequest(200, "Specify token_address for custom url.")
```

Comment:
I think it would be better to use standard python error here, or write a custom one instead of using internal errors from cairo lang library.